### PR TITLE
Update cdn url

### PR DESCRIPTION
**Pull request by MeisterJustice**
- Add an optional parameter on the initialization  - cdnUrl
- Add a hook containing a state that stores the new production cdn
- check if user passed cdnUrl on initialization parameter
   - if yes, update the stored cn to the new one

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { useLazerpay } from './dist/index'
 
 const App = () => {
   const config = {
-    publicKey: 'pk_live_0N24k7lsrr7NGfrDQpIjPGy9z61LkXjUqxX3r99XblXHemwMht',
+    publicKey: 'pk_live_0N24k7lsrr7NGfrDQpIjPGy9z61LkXjUqxX3r99XblXHemwMht', // live key
```
Comment (Frontend-io) [inline example/src/App.tsx:7]: Can we put this in .env instead?

File: src/useCdn.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
+
+export default function useCdn(): [string, (cdn: string) => void] {
```
Comment (Frontend-io) [inline src/useCdn.ts:3]: Just wondering. There isnt really much logic to this hook, i kinda think we can just perform the update in the file itself?

**Review by Frontend-io — CHANGES_REQUESTED** [review]

File: src/useCdn.ts (hunk 2 above)
Comment (MeisterJustice) [reply src/useCdn.ts:3]: The update isn't made on the file; `scripts.js` but on the `useLazerpay` file.

File: example/src/App.tsx (hunk 1 above)
Comment (MeisterJustice) [reply example/src/App.tsx:7]: uhmm do you think it's really necessary? This is meant to be just an example app (file) anyone can look at

Comment (Frontend-io) [reply example/src/App.tsx:7]: Ok ok

**Review by Frontend-io — APPROVED** [review]
Okay. I think we can merge anyway, it’s not a big deal.

**Pull request merged**